bulma-rtl.sass

```diff
@@ -0,0 +1,10 @@
+/*! bulma.io v0.4.0 | MIT License | github.com/jgthms/bulma */
+@charset "utf-8"
+
+$right: left
+$left: right
+
+html
+  direction: rtl
```

Comment:
thanks for reviewing @mightyiam .

moving this into base modules then I should apply @if to add this directional style only when we are making an rtl version. but if you want I can do that.